airbyte-integrations/connectors/source-relational-db/src/main/java/io/airbyte/integrations/source/relationaldb/AbstractRelationalDbSource.java

```diff
@@ -38,6 +38,20 @@ public AutoCloseableIterator<JsonNode> queryTableFullRefresh(final Database data
         getFullTableName(schemaName, tableName)));
   }
 
+  @Override
+  public AutoCloseableIterator<JsonNode> queryTableFullRefresh(final Database database,
```

Comment:
This function is defined twice in this file, did you mean to edit the one above?

Reply:
see above